nixos/modules/security/pam.nix

```diff
@@ -7,6 +7,7 @@ with lib;
 
 let
 
+  moduleArgsType = with types; attrsOf (nullOr (oneOf [ bool str int pathInStore ]));
```

Comment:
nitpick: `moduleSettingsType` would be more consistent with the naming in the rest of this module. It uses `args` to refer to the `module-arguments` concept defined by PAM, and `settings` for the NixOS abstraction of key-value pairs.